### PR TITLE
Lucee 5 references and updating the README

### PR DESCRIPTION
**Pull request by andyj**
Updated the install script to look at Lucee 5, and a more recent version of Java. 

Also changed the README to download to ~/ instead of /root/

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 #configuration options
-export LUCEE_VERSION="4.5.1.022"
+export LUCEE_VERSION="5.0.0.45"
```
Comment (SimonHooker) [inline install.sh]: Would bump this to 5.0.0.252

File: etc/nginx/lucee.conf [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 #block the lucee-context except for certain ip
 location ~* /lucee/ {
-	allow 10.0.0.10;
+	allow 173.162.204.40/30;
```
Comment (pfreitag) [inline etc/nginx/lucee.conf]: We will want to change this back to something like 10.0.0.10

File: etc/lucee/lucee-server.xml [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+Path placeholders: 
+	{lucee-server}: path to the Lucee server directory typical where the Lucee.jar is located
+	{lucee-config}: same as {lucee-server} in server context and same as {lucee-web} in web context}
+	{temp-directory}: path to the temp directory of the current user of the system
+	{home-directory}: path to the home directory of the current user of the system
+	{web-root-directory}: path to the web root
+	{system-directory}: path to thesystem directory
+	{web-context-hash}: hash of the web context
+
+--><cfLuceeConfiguration hspw="92e290fdf0db12459894e2244bb698fce60d72b7127d53a4e96ab752f50bbd98" salt="8F5C3B30-0011-4FE3-842733CB26D0F3DC" version="5.0">
```
Comment (pfreitag) [inline etc/lucee/lucee-server.xml]: Should we generate the salt/password randomly?

**Comment by pfreitag** [comment]
@andyj thanks for all the work here -- one thing that is holding me back from merging, besides doing some testing is that it appears to be using the viviotech installer which includes a bundled version of Tomcat. While the installer is great, I don't like that I have to update tomcat manually. That is why this repo uses the Ubuntu `tomcat7` package, so you can update it using `apt-get`

**Comment by andyj** [comment]
Bin this pull request. I like your approach about different branches for
different setups I.e 16.04 LTS support.

Happy to help test new development out though.

**Comment by pfreitag** [comment]
Closing this as requested.

**Pull request closed without merging**